agreements/create_trading_organisations.sql

```diff
@@ -0,0 +1,9 @@
+/*
+Title       create_trading_organisations.sql
+Date        July 19th 2020
+Author      Trevor Cummings
+Description This file is a script to create data for trading_organisations. This data eventually will reside in Conclave     
+*/
+INSERT INTO public.trading_organisations(
+	trading_organisation_id, trading_organisation_type_id, organisation_id, trading_organisation_name)
```

Comment:
`trading_organisation_type_id` does not exist on `trading_organisation`

Reply:
Hi @adrianmilne , I just looked at the code in github and trading organisations doesn't have trading_organisation_type_id anymore. Have you got an old version of the code or am I looking in a different branch.

Reply:
@trevorcummingsCCS - my comment was from June last year - so probably safe to say an old version of the code 😄